src/models/order.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require("mongoose");
+const schema = mongoose.Schema;
+
+const orderSchema = new schema({
+  userId: {
+    type: schema.Types.ObjectId,
+    ref: "user",
+  },
+  products: [
+    {
+      productId: {
+        type: schema.Types.ObjectId,
+        ref: "product",
+      },
+      quantity: {
+        type: Number,
+        default: 1,
+      },
+    },
+  ],
+  amount: {
+    type: Number,
+    require: true,
+  },
+});
+
+module.exports = mongoose.model("Order", orderSchema);
```

Comment:
Should follow the consistent pattern for all model's name

Reply:
resolved